elfloader-tool/src/plat/bcm2711/smp.c

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2019, ARM Ltd
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ *
+ */
+
+#include <autoconf.h>
+#include <elfloader/gen_config.h>
+#include <types.h>
+
+#if CONFIG_MAX_NUM_NODES > 1
+#include <types.h>
+#include <elfloader.h>
+#include <armv/machine.h>
+#include <armv/smp.h>
+#include <printf.h>
+#include <abort.h>
+
+#define MAX_CORES 4
+
+extern void core_entry_head(unsigned long stack);
+
+unsigned long core_stack;
```

Comment:
If this holds an address, then we should simply say so.
```suggestion
void *core_stack;
```